packages/functions/transform/src/set-merge-requests.test.ts

```diff
@@ -34,6 +34,10 @@ beforeAll(async () => {
 
   await migrate(extractDb, { migrationsFolder: "../../../migrations/extract" });
   await migrate(transformDb, { migrationsFolder: "../../../migrations/transform" });
+  await extractDb.insert(extract.namespaces).values([
+    { id: 1, externalId: 2000, forgeType: 'github', name: 'crocoder-dev' }
+  ]);
+
```

Comment:
ejecting from style defined by this test with beforeEach, not sure if that's okay